TTS/bin/compute_statistics.py

```diff
@@ -14,35 +14,50 @@
 
 
 def main():
-    """Run preprocessing process."""
-    parser = argparse.ArgumentParser(description="Compute mean and variance of spectrogtram features.")
+    """Compute feature statistics for normalization."""
+    parser = argparse.ArgumentParser(
+        description="Compute mean and variance of spectrogtram features."
+        )
```

Comment:
I've reformatted some of the lines that are a lot longer than 80 chars.

I was also trying to deindent the ending parenthesis to be inline with the rest of the code, but the editor got the best of me here I see ...

Reply:
The line limit is 119 which is the default of github. I think 80 is too short on today's resolution standards.

Reply:
Yeah, but when you open up vertical splits and all it gets messy :slightly_smiling_face:

Reply:
don't open up :) or you can edit the line length in your IDE too

Reply:
What do you use as an IDE ?

Reply:
Ahah :smile: 

Still, even if viewing one file you do need other things on screen. In Vim I have a file browser and an outline, as well as IPython in a side by side terminal for some debugging. In Spyder these are all integrated in the IDE.

Another thing is screen size - if you're on a laptop there's not much screen real-estate to work with. Even with HiDPI support - I do have to zoom substantially to get the code readable :stuck_out_tongue: 

Maybe it's a good idea for a discussion topic, although I'm not too optimistic about people backing me up :smile:

Reply:
Yeah feel free to open up a discussion. If people think like you I am totally okay to change the line length.

Reply:
Maybe :+1: Seems that Black defaults to 88 characters per line, in line with 79 being a bit too restrictive and 90ish being recommended in [Beyond PEP 8](https://youtu.be/wf-BqAjZb8M?t=260). Good video on PEP8 zealots btw :slightly_smiling_face: